src/amber/router/cookies.cr

```diff
@@ -0,0 +1,228 @@
+require "http"
```

Comment:
This is awesome!

What do you think of submitting this to the Crystal standard library as an enhancement to the existing Cookies store? https://crystal-lang.org/api/0.22.0/HTTP/Cookies.html

Or a separate shard that can be leveraged by other frameworks like kemal?